src/pages/JobView/UpdateJobButton.tsx

```diff
@@ -85,10 +61,20 @@ const UpdateJobButton = ({ job }: UpdateJobButtonProps) => {
 
         fetchSystemCommandPair()
 
+        const getJobTemplate = (job: Job) => ({
+            id: job.id,
+            misfire_grace_time: job.misfire_grace_time,
+            trigger: job.trigger,
+            name: job.name,
+            request_template: job.request_template,
+            trigger_type: job.trigger_type,
+            timeout: job.timeout,
+            coalesce: job.coalesce,
+            max_instances: job.max_instances,
+          })
+
         const onClick: MouseEventHandler<HTMLButtonElement> = () => {
-          setIsReplay(true)
-          setIsJob(true)
-          setRequestModel(extractRequestModel(job))
+          setJob(getJobTemplate(job))
```

Comment:
I was using only a subset here because I was only getting values that the page to update would care about for the form values. I changed how it works some so the update page determines which values to use instead